ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/db-details/connection-details.tsx

```diff
@@ -0,0 +1,67 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Messages } from '../../cluster-overview.messages';
+import OverviewSection from '../../overview-section';
+import { ConnectionDetailsOverviewCardProps } from '../card.types';
+import OverviewSectionRow from '../../overview-section-row';
+import { Box, Typography } from '@mui/material';
+import { CopyToClipboardButton } from '@percona/ui-lib';
+import { HiddenPasswordToggle } from 'components/hidden-row';
+
+export const ConnectionDetails = ({
+  loading,
+  hostname = '',
+  username,
+  password,
+  port,
+}: ConnectionDetailsOverviewCardProps) => {
+  return (
+    <OverviewSection
+      title={Messages.titles.connectionDetails}
+      loading={loading}
+    >
+      <OverviewSectionRow
+        label={Messages.fields.host}
+        content={hostname.split(',').map((host) => (
+          <Box key={host} sx={{ display: 'flex', gap: 1 }}>
+            <Typography variant="body2">{host}</Typography>
+            <CopyToClipboardButton
+              buttonProps={{
+                color: 'primary',
+                size: 'xs',
```

Comment:
```suggestion
                size: 'small',
```

Reply:
@dianabirs we have new sizes for button, it's not our usual small size and mui small. peak icon = 20*20, overview icons = 16*16
![image](https://github.com/user-attachments/assets/0311ff43-0ead-4545-866c-7402bacab75a)
![image](https://github.com/user-attachments/assets/f9295035-8ca3-44b3-a7ca-fb6a19f60473)

Reply:
@dianabirs as we talked tomorrow, I returned all sizes to current small sizes and will create separate PR to fix buttons and icons sizes
![image](https://github.com/user-attachments/assets/b65633dc-eff2-45fc-b8a1-6d829415e12b)